xapian-applications/omega/Makefile.am

```diff
@@ -178,6 +178,10 @@ omindex_LDADD = $(MAGIC_LIBS) $(XAPIAN_LIBS) $(ZLIB_LIBS) libutf8convert.la
 omindex_pdf_SOURCES = assistant.cc worker_comms.cc common/str.cc handler_poppler.cc
 omindex_pdf_LDADD = $(POPPLER_LIBS) $(XAPIAN_LIBS)
 
+omindex_ebook_SOURCES = assistant.cc worker_comms.cc handler_libebook.cc
+omindex_ebook_LDADD = $(LIBEBOOK_LIBS) $(XAPIAN_LIBS)
+omindex_ebook_CXXFLAGS = $(LIBEBOOK_CFLAGS) $(XAPIAN_CXXFLAGS)
+
```

Comment:
As I am using stringutils.h, I add XAPIAN_CXXFLAGS. Is it okay?

Reply:
You shouldn't need it (not `XAPIAN_LIBS`).

Most of `stringutils.h` is macros and inline functions, which don't require anything to be linked.

If you were using `C_tolower()`, `C_isdigit()` (which are like the standard library versions except they always work in the "C" locale rather than returning results which depend on the current locale), etc then you would need to link the library, as they use a data table that is provided by it.  But you aren't using any of these that I can see.

Reply:
Oh i see, `XAPIAN_CXXFLAGS` is needed so that `xapian/constinfo.h` can be included by `stringutils.h`.

Probably we should split `stringutils.h` into two headers to avoid that being necessary, but for now keeping this is OK.